db/1.8/modules/ROOT/pages/database-connector-studio.adoc

```diff
@@ -1,32 +1,107 @@
+You can configure one of these input sources to use with Database Connector:
+
+* *Database > On Table Row* +
+Initiates a flow by selecting from a table at a regular interval and generates one message per obtained row
+* *HTTP > Listener* +
+Initiates a flow each time it receives a request on the configured host and port
+* *Scheduler* +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an *On Table Row* source, follow these steps:
+
+. In the *Mule Palette* view, select *Database > On Table Row*.
+. Drag *On Table Row* to the Studio canvas.
+. On the *On Table Row* configuration screen, optionally change the value of the *Display Name* field.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the source in the app.
+. On the *Database Config* window, for *Connection*, select any of the connection types to provide to this configuration.
+. On the *General* tab, specify the database connection information for the connector.
+. On the *Transactions* tab, optionally specify the transaction isolation, or XA transactions when connecting to the database.
+. On the *Advanced* tab, optionally specify connection pooling and reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified database.
+. Click *OK* to close the window.
+. On the *On Table Row* configuration screen, in *Table*, specify the name of the table to select from.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Database Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Database Connector* and then select the desired operation.
+. Drag the operation onto the Studio canvas and to the right of the input source.
+
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Database Connector can connect to any database for which a JDBC driver is available. To make it easier to connect to the most popular databases (such as MySQL, Oracle, and Microsoft SQL Server), the connector includes a series of different connection types.
+
+To configure the global element for Database Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *Database Config* window, for *Connection*, select any of the connection types to provide to this configuration.
+. On the *General* tab, specify the database connection information for the connector.
```

Comment:
For the connection configuration, we usually list the available connection types--in this case:
Derby
Generic
Microsoft SQL
MySQL
Oracle
etc